i18n/translations/de-AT.json

```
@@ -0,0 +1,138 @@
+    "tab": {
+        "uplinks": "Uplinks",
+        "versions": "Versionen",
+        "dependencies": "Abhängigkeiten",
+        "readme": "Liesmich"
+    },
+    "uplinks": {
+        "title": "Uplinks",
+        "no-items": "{{name}} hat keine Uplinks."
+    },
+    "versions": {
+        "current-tags": "Aktuelle Tags",
+        "version-history": "Versionsgeschichte",
+        "not-available": "Nicht verfügbar"
+    },
+    "package": {
+        "published-on": "Veröffentlicht am {{time}} •",
+        "version": "v{{version}}",
+        "visit-home-page": "Auf die Homepage gehen",
+        "homepage": "Homepage",
+        "open-an-issue": "Einen Fehler melden",
+        "bugs": "Fehler",
+        "download": "{{what}} herunterladen",
+        "the-tar-file": "die tar-Datei",
+        "tarball": "Tarball"
+    },
+    "dependencies": {
+        "has-no-dependencies": "{{package}} hat keine Abhängigkeiten",
+        "dependency-block": "{{package}}@{{version}}"
+    },
+    "form": {
+        "username": "Benutzername",
+        "password": "Passwort"
+    },
+    "form-placeholder": {
+        "username": "Dein Benutzername",
+        "password": "Dein sicheres Passwort"
+    },
+    "form-validation": {
+        "required-field": "Dieses Feld ist erforderlich",
+        "required-min-length": "Dieses Feld erfordert die Mindestlänge von {{length}}",
+        "unable-to-sign-in": "Anmeldung nicht möglich",
+        "username-or-password-cant-be-empty": "Benutzername und Passwort dürfen nicht leer sein!"
+    },
+    "help": {
+        "title": "Noch kein Paket publiziert.",
+        "sub-title": "Um dein erstes Paket einfach zu publizieren:",
+        "first-step": "1. Einloggen",
+        "first-step-command-line": "npm adduser --registry {{registryUrl}}",
+        "second-step": "2. Publizieren",
+        "second-step-command-line": "npm publish --registry {{registryUrl}}",
+        "third-step": "3. Diese Seite aktualisieren."
+    },
+    "sidebar": {
+        "detail": {
+            "latest-version": "Letzte v{{version}}",
+            "version": "v{{version}}"
+        },
+        "installation": {
+            "title": "Installierung",
+            "install-using-yarn": "Mit yarn installieren",
+            "install-using-yarn-command": "yarn add {{packageName}}",
+            "install-using-npm": "Mit npm installieren",
+            "install-using-npm-command": "npm install {{packageName}}",
+            "install-using-pnpm": "Mit pnpm installieren",
+            "install-using-pnpm-command": "pnpm install {{packageName}}"
+        },
+        "repository": {
+            "title": "Repository"
+        },
+        "author": {
+            "title": "Autor(in)"
+        },
+        "distribution": {
+            "title": "Neueste Distribution",
+            "license": "Lizenz",
+            "size": "Größe",
+            "file-count": "Anzahl der Dateien"
+        },
+        "maintainers": {
+            "title": "Betreuerinnen"
```

```suggestion
            "title": "Maintainer"
```

In IT it's rather usual to talk about "maintainers" even in German.